### ISSUE TITLE
RimWorld 1.5

### ISSUE THREAD

**Issue by Zetrith** [rwmt/Multiplayer#429]
An issue tracking Multiplayer's support of RW 1.5. There's going to be a separate issue for Anomaly when it releases but we can make notes here.

- [ ] Starting rituals needs a rework
- [ ] Reading policies
- [ ] `Policy` renaming
- [ ] ISlotGroups need a sync worker (related to #412)
- [ ] Storage groups were added in some previous update and could be made per-faction

Note that the new multithreaded renderer is not an issue as it never executes in parallel with the simulation.

**Comment by SokyranTheDragon** [comment]
`PawnColumnWorker_Designator:SetValue` and `PawnColumnWorker_Sterilize:SetValue` syncing will need to be reworked. These two will now display a confirmation dialog under specific circumstances, and work as they previously did under others. [Sync methods declared here.](https://github.com/rwmt/Multiplayer/blob/5b7a8ea9da3a85bdc78cbe3471cb979f794c0131/Source/Client/Syncing/Game/SyncMethods.cs#L86-L89)

**Comment by SokyranTheDragon** [comment]
It looks like we'll need a session and patches for `Dialog_EnterPortal`. Looks like it may be a non-Anomaly feature (doesn't seem like there's DLC checks on it).

**Comment by twothe** [comment]
Having severe issues with caravans on encounter maps since 1.5.
- Caravan reform turns into forming a completely new carvan, as in: they have to unload and re-load everything
- Constant desyncs, especially if members got wounded

**Comment by Knniff** [comment]
I had problems when trying to capture anomalies as well as trying to do a role change. When trying to setup the role change ritual it was impossible to drag the respective colonists to their roles(spectator/the one receiving the role).

**Comment by SokyranTheDragon** [comment]
It seems that stopping research is not synced (along with the 2 dev mode gizmos, which I'm pretty sure were reported ages ago).

**Comment by mantis** [comment]
> It seems that stopping research is not synced (along with the 2 dev mode gizmos, which I'm pretty sure were reported ages ago).

Have been hitting this - you (@SokyranTheDragon) reported it @ https://github.com/rwmt/Multiplayer/issues/290 - assuming there might also be something to do to handle anomalyKnowledge and currentAnomalyKnowledgeProjects that exist
